src/main/java/Parser.java

```diff
@@ -3,7 +3,6 @@
 public class Parser {
     public static String getFirstWord(String line) {
         String firstWord;
-        //line = line.trim();
         int firstSpace = line.indexOf(' ');
         if (firstSpace != -1) {
```

Comment:
Regarding the coding quanlity, comments should be given regarding -1, otherwise, it is not meaningful.